src/DynamoPackages/PackageManagerExtension.cs

```diff
@@ -339,10 +342,9 @@ internal bool CheckIfPackagesTargetOtherHosts(IEnumerable<PackageVersion> newPac
             var containsPackagesThatTargetOtherHosts = false;
             //fallback list of hosts as of 9/8/23
             IEnumerable<string> knownHosts =  new List<string> { "Revit", "Civil 3D", "Alias", "Advance Steel", "FormIt" };
-            //if analytics is disabled, treat this as network black out
-            //and don't ask dpm for known hosts.
-            //TODO we currently use this like a no network comms flags - work on introducing a new flag or renaming this flag.
-            if (!Dynamo.Logging.Analytics.DisableAnalytics)
+
+            //we don't ask dpm for known hosts in offline mode.
+            if (!noNetworkMode)
             {
                 // Known hosts
                 knownHosts = PackageManagerClient.GetKnownHosts();
```

Comment:
I think that for 2.19.1 I implemented something similar but also defaulted to some known hosts, can we unify that logic? I can't recall where though.

Reply:
This is what I was referring to: https://github.com/DynamoDS/Dynamo/compare/RC2.19.0_master..RC2.19.1_master (a comparison between 2.19.1 and 2.19.0). The changes that went into 2.19.1 somehow also seemed to be present in master so I didn't need to do anything extra. 

I haven't changed the logic other than replacing `DisableAnalytics` with `noNetworkMode`. Please let me know if I'm missing something.

Reply:
Looks like the defualts are in this PR
`IEnumerable<string> knownHosts = new List<string> { "Revit", "Civil 3D", "Alias", "Advance Steel", "FormIt" };`